wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallController/ActiveCallRouter.swift

```diff
@@ -39,7 +44,10 @@ protocol CallQualityRouterProtocol: AnyObject {
     func presentCallQualityRejection()
 }
 
+typealias PostCallAction = ((@escaping Completion) -> Void)
```

Comment:
opinion: I always forget the types and thus prefer not to have type alias to look up, but just write the definition in place to read simply:
```swift
var scheduledPostCallAction: PostCallAction? // now
var scheduledPostCallAction: ((@escaping Completion) -> Void)? // preferred
```

Reply:
so you never use typealias ?

Reply:
I do for some cases, but not for closures..
A required case could be that you have an associated type or generics.